hack/istio/multicluster/install-multi-primary.sh

```diff
@@ -17,13 +17,14 @@ source ${SCRIPT_DIR}/env.sh $*
 install_istio() {
   local clustername="${1}"
   local network="${2}"
+  local pf_yaml="${3}"
   if [ ! -z "${ISTIO_TAG}" ]; then
     local image_tag_arg="--image-tag ${ISTIO_TAG}"
   fi
   if [ ! -z "${ISTIO_HUB}" ]; then
     local image_hub_arg="--image-hub ${ISTIO_HUB}"
   fi
-  "${ISTIO_INSTALL_SCRIPT}" ${image_tag_arg:-} ${image_hub_arg:-} --client-exe-path "${CLIENT_EXE}" --cluster-name "${clustername}" --istioctl "${ISTIOCTL}" --istio-dir "${ISTIO_DIR}" --mesh-id "${MESH_ID}" --namespace "${ISTIO_NAMESPACE}" --network "${network}"
```

Comment:
@nrfox by deleting this, we lost support of setting up things the istio/istioctl way, so two questions? 

1. Do we want to keep both functionalities ? 
2. If so, which one should be default?

Reply:
Yes that's the goal with these changes to move from installing istio with istioctl to installing istio with sail. At the end of the day its all istio and how we install it only matters in how much work it is to maintain but shouldn't affect how Kiali tests with istio.